src/classes/TDTM_TriggerHandler.cls

```diff
@@ -73,7 +73,7 @@ public class TDTM_TriggerHandler {
 
         TDTM_Runnable.Action thisAction = TDTM_TriggerActionHelper.determineTriggerAction(isBefore,
                                                       isAfter, isInsert, isUpdate, isDelete, isUnDelete);
-        UTIL_Debug.debugWithInfo('****ACTION: ' + thisAction + ' on ' + describeObj.getLabel());
+        UTIL_Debug.debug(LoggingLevel.ERROR, '****ACTION: ' + thisAction + ' on ' + describeObj.getLabel());
```

Comment:
Good catch. No real reason. I had changed in my debug test just so it would show up in the different levels, but it should match the other debug statements in this method.